### PR TITLE
Enabled Pagination

### LINKED ISSUE #7084: [UI] Pagination not working on MesheryDesigns and MesheryApplication
#### Current Behavior

Pagination isn't working on MesheryDesigns and MesheryApplication

#### Screenshots/Logs

https://user-images.githubusercontent.com/103871275/218848927-4c0c6e3a-c6a8-4a98-bff4-fc468344008f.mp4

#### Environment

- Browser: Firefox
- Host OS: Linux
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by hs05june**
**Notes for Reviewers**

This PR fixes #7084 

https://user-images.githubusercontent.com/103871275/218850948-f412484b-2bc7-46fb-b91f-ed8a33761816.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryFilters/FiltersGrid.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ function FiltersGrid({ filters=[],handleDeploy, handleUndeploy, handleClone, han
         isDelete={!modalOpen.deploy}
         title={ modalOpen.name }
         componentCount = {modalOpen.count}
-        tab={modalOpen.deploy ? 0 : 1}
+        tab={modalOpen.deploy ? 2 : 1}
```
Comment (leecalcote) [inline ui/components/MesheryFilters/FiltersGrid.js:144]: I'm ignorant here. What does "2" signify here?

**Review by leecalcote — APPROVED** [review]
LGTM

File: ui/components/MesheryFilters/FiltersGrid.js (hunk 1 above)
Comment (hs05june) [reply ui/components/MesheryFilters/FiltersGrid.js:144]: 2 is the tab number for deploy in the confirmation modal. 
Screenshot 2023-02-16 104303
Actually, it is a change made in my previous PR.

**Comment by hs05june** [comment]
@leecalcote Really a proud moment for me. It's my 10th PR that got merged in meshery. Have learnt much and will continue to learn.

**Pull request merged**